tests/reporter/kubernetes.go

```diff
@@ -567,6 +569,55 @@ func (r *KubernetesReporter) logJournal(virtCli kubecli.KubevirtClient, logsdir
 	}
 }
 
+func (r *KubernetesReporter) logMultus(virtCli kubecli.KubevirtClient, path string, nodes *v1.NodeList, since time.Time) {
+	if path == "" {
+		fmt.Fprintf(os.Stderr, "target path is invalid %q, skipping Multus log dump\n", path)
+		return
+	}
+
+	const logFilePath = "/proc/1/root/var/log/multus.log"
+	checkLogFileExistsCmd := []string{"/usr/bin/sh", "-cx", "[ -f " + logFilePath + " ]"}
+	dumpContentCmd := []string{"/usr/bin/cat", logFilePath}
+
+	for _, node := range nodes.Items {
+		pod, err := kubecli.NewVirtHandlerClient(virtCli).Namespace(flags.KubeVirtInstallNamespace).ForNode(node.Name).Pod()
+		if err != nil {
+			fmt.Fprintf(os.Stderr, failedGetVirtHandlerPodFmt, node.Name, err)
+			return
+		}
+
+		if _, _, err = tests.ExecuteCommandOnPodV2(virtCli, pod, virtHandlerName, checkLogFileExistsCmd); err != nil {
+			return
+		}
+
+		stdout, _, err := tests.ExecuteCommandOnPodV2(virtCli, pod, virtHandlerName, dumpContentCmd)
```

Comment:
How big will the log files be, since we are dumping them into memory here? I don't think that this will be a problem wrt the current config, but in general I'd rather use a buffered approach for this.